pkg/mermaid/mermaidintegrationdiagram.go

```diff
@@ -0,0 +1,88 @@
+package mermaid
```

Comment:
File name could probably be shortened to integration or integration_diagram?
The package name already indicates well enough it uses mermaid imo

Reply:
Agreed. Will change this.